### ISSUE TITLE
Add more topics and resources

### ISSUE THREAD

**Issue by aviaryan** [aviaryan/learn-for-free#1]
Please add more topics, sub-topics, and resources to grow this list. Please make sure to read [CONTRIBUTING.md](https://github.com/aviaryan/learn-for-free/blob/master/CONTRIBUTING.md) before opening a pull request.

**Comment by NavneetNandan** [comment]
@aviaryan Should I add data science resources? Also, is it fine to add the courses whose certifications are paid but learning is free?

**Comment by aviaryan** [comment]
@NavneetNandan Yes and yes. I think you will have to create a new category for Data Science since it doesn't go in Programming.

**Comment by NavneetNandan** [comment]
@aviaryan Yes, I agree does not go in Programming. I also think it is big enough to have a separate category in it. It could have sub topics like Deep Learning, Natural Language Processing, Computer Vision etc.

**Comment by aviaryan** [comment]
@NavneetNandan Agreed.

**Comment by dhruvbpatel** [comment]
I have a query, our goal is to provide free resources for learning various topics right, but we can't add cracked/pirated paid courses right?

**Comment by aviaryan** [comment]
@dhruvbpatel No

**Comment by dhruvbpatel** [comment]
okay thanks for clarifying 👍
